app/ui.js

```diff
@@ -502,7 +502,6 @@ var UI;
             }
 
             UI.enableDisableViewClip();
-            document.getElementById('noVNC_setting_resize').disabled = UI.connected;
```

Comment:
Yes, you are right.. I'm not sure why I thought that was acceptable tbh. I'll take a stab at fixing these things